src/ageml/ui.py

```diff
@@ -1761,7 +1800,9 @@ def model_age_command(self):
         print("Hyperparameter tuning. Number of points in grid search: (Default: 0)")
```

Comment:
Shoudl the default if we are actually running hypeparameter tunning not be at least 2? Something maybe like 5? Is not it should be saying hyperparameter tunning.

Reply:
True, missed this